src/App.js

```diff
@@ -1,13 +1,59 @@
 import React from 'react';
 
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+constructor() {
+  super();
+  this.state = {
+
+    todos: [
+
+      {
+
+        task: 'Organize Garage',
+        id: 1528817077286,
+        completed: false
+
+      },
+      {
+
+        task: 'Bake Cookies',
+        id: 1528817084358,
+
+        completed: false
+      }
+      ],
+
+      todo: ''
+
+    };
+  }
+
+  changeTodo = e => this.setState({ [e.target.name]: e.target.value });
+
+  addTodo = e => {
+
+    e.preventDefault();
+    const newTodo = { task: this.state.todo, completed: false, id: Date.now() };
+    this.setState({
+      todos: [...this.state.todos, newTodo],
+      todo: ''
+
```

Comment:
good job resetting the todo state field to an empty string after submitting the form and remembering to prevent the form from reloading on submission.